app/src/routes/v1.js

```diff
@@ -33,6 +35,9 @@ router.get('/api-spec.yaml', (_req, res) => {
 /** Health Router */
 router.use('/health', keycloak.protect(), healthRouter);
 
+/** File Types Router */
+router.use('/fileTypes', keycloak.protect(), fileTypesRouter);
+
```

Comment:
Style: I suggest this be above the health router for alphabetical ordering reasons.